### ISSUE TITLE
Server-side Disabling of Chunkloader

### ISSUE THREAD

**Issue by ghost** [ReikaKalseki/Reika_Mods_Issues#1194]
DragonAPI is booting players who have a different setting in RotaryCraft.cfg than the server does.

Context:
A server is being set up using Survival Industry, a modpack containing RotaryCraft.  The server operators want to disable the RotaryCraft chunkloader.  However, every player who connects to the server (without manually changing the config) is booted by DragonAPI with the following error:

```
DRAGONAPI: Client <playername> attempted to join with a Critical config mismatch. Config Entry: ROTARYCRAFT - "Enable Chunk Loader", Server Value=0, Client Value=1. Login cannot continue.;
```

In other words, DragonAPI is blocking players to a server that wants to limit chunkloading, simply because the modpack has it enabled by default.  Is there a way to fix this without disabling the chunkloader in the base modpack?

**Comment by ReikaKalseki** [comment]
No, because configs like that have to match.
